aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -1826,6 +1873,29 @@ func extractCloudWatchLoggingConfiguration(s3 map[string]interface{}) *firehose.
 
 }
 
+func extractVpcConfiguration(es map[string]interface{}) *firehose.VpcConfiguration {
+	config := es["vpc_config"].([]interface{})
+	if len(config) == 0 {
+		return nil
+	}
+	vpcConfig := config[0].(map[string]interface{})
+	s := vpcConfig["subnet_ids"].(*schema.Set).List()
```

Comment:
helper func `expandStringSet()` can be used on `vpcConfig["subnet_ids"].(*schema.Set)` instead